src/theory/strings/core_solver.cpp

```diff
@@ -2121,7 +2125,8 @@ void CoreSolver::processDeq(Node ni, Node nj)
           Node uxLen = nm->mkNode(STRING_LENGTH, ux);
           Node uyLen = nm->mkNode(STRING_LENGTH, uy);
           std::vector<Node> newSkolems;
-          Node conc = getDecomposeConclusion(ux, uyLen, false, skc, newSkolems);
+          Node conc =
+              getDecomposeConclusion(ux, uyLen, false, true, skc, newSkolems);
```

Comment:
I believe this should be `false, options::stringLenConc()` instead of `false, true`, right?

Reply:
As discussed offline, I've added a clarifying comment about this.